core/services/chainlink/application.go

```diff
@@ -71,7 +71,7 @@ type Application interface {
 	GetWebAuthnConfiguration() sessions.WebAuthnConfiguration
 
 	GetExternalInitiatorManager() webhook.ExternalInitiatorManager
-	GetChainSet() evm.ChainSet
+	GetChains() Chains
```

Comment:
On second thoughts maybe this name is ok

Reply:
I shortened it from `ChainSets` to `Chains` when it held ORM fields too, but I thought it still seemed cleaner with only `ChainSets` - plus it could grow.